lms/djangoapps/discussion_api/api.py

```diff
@@ -647,9 +651,11 @@ def update_thread(request, thread_id, update_data):
     # Only save thread object if some of the edited fields are in the thread data, not extra actions
     if set(update_data) - set(actions_form.fields):
         serializer.save()
-        thread_edited.send(sender=None, user=request.user, post=cc_thread)
     api_thread = serializer.data
     _do_extra_actions(api_thread, cc_thread, update_data.keys(), actions_form, context)
+
+    # signal to update Teams information
+    thread_edited.send(sender=None, user=request.user, post=cc_thread)
```

Comment:
What is the rationale for moving this out?
Note: now, the signal will be sent even if there are no changes.